pallets/cf-reward/src/lib.rs

```diff
@@ -0,0 +1,208 @@
+// Copyright 2019-2021 Centrifuge Inc.
+// This file is part of Cent-Chain.
+
+// Cent-Chain is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cent-Chain is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cent-Chain.  If not, see <http://www.gnu.org/licenses/>.
+
+//! # Reward module for Centrifuge Parachain Campaigns
+//!
+//! ## Overview
+//! The function of this pallet is to provide the centrifuge specific reward functionality for
+//! contributors of the relay chain crowdloan. In order to provide this functionality the pallet
+//! implements the `Reward` Trait from the `Claim` Pallet.
+//! Before any rewards can be provided to contributors the pallet MUST be initialized, so that the
+//! modules account holds the necessary funds to reward.
+//! All rewards are payed in form of a `vested_transfer` with a fixed vesting time. The vesting time
+//! is defined via the modules `Config` trait.
+//!
+//! ## Callable Functions
+//!
+//! - `initialize` - Initializes the module by transfering founds to the modules account and activating an init-lock
+//!
+//! ## Implementations
+//!
+//! - [`Reward`](crowdloan_claim::reward::Reward) : Rewarding functionality for contributors in a relay
+//!   chain crowdloan.
+
+// Ensure we're `no_std` when compiling for Wasm.
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use frame_support::{decl_module, decl_storage, decl_event, decl_error, weights::{Weight}, Parameter, ensure};
+use frame_system::{ensure_root, RawOrigin};
```

Comment:
Seems easier to read when packages are in alphabetic order and no need for braces when we import only one package. Moreover, it seems also more readable when  not using one line only.

```rust
use frame_support::{
  decl_module, decl_storage, decl_event, decl_error, 
  ensure,
  Parameter, 
  weights::Weight, 
};
```

Reply:
I think ordering them alphabetically is too big of an overhead. Splitting it into multi-lines is fine.

Reply:
> I think ordering them alphabetically is too big of an overhead. Splitting it into multi-lines is fine.

We should use a Linter for that.